tarantool/src/main/java/com/yahoo/ycsb/db/TarantoolClient.java

```diff
@@ -0,0 +1,168 @@
+package com.yahoo.ycsb.db;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.List;
+import java.util.Arrays;
+import java.util.Vector;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.HashMap;
+import java.util.Properties;
+
+import java.io.IOException;
+
+import org.tarantool.TarantoolConnection16;
+import org.tarantool.TarantoolConnection16Impl;
+import org.tarantool.TarantoolException;
+import org.tarantool.CommunicationException;
+
+import com.yahoo.ycsb.DB;
+import com.yahoo.ycsb.DBException;
+import com.yahoo.ycsb.ByteIterator;
+import com.yahoo.ycsb.StringByteIterator;
+
+public class TarantoolClient extends DB {
+	
+	public static final String HOST_PROPERTY  = "tarantool.host";
+	public static final String PORT_PROPERTY  = "tarantool.port";
+	public static final String SPACE_PROPERTY = "tarantool.space";
+
+	public static final String DEFAULT_HOST   = "localhost";
+	public static final int    DEFAULT_PORT   = 3301;
+	public static final int    DEFAULT_SPACE  = 1024;
+	
+	private static final Logger log = Logger.getLogger(TarantoolClient.class.getName());
+	private TarantoolConnection16Impl connection;
+	private int spaceNo;
+
+	public void init() throws DBException {
+		Properties props = getProperties();
+
+		int port = DEFAULT_PORT;
+		String portString = props.getProperty(PORT_PROPERTY);
+		if (portString != null) {
+			port = Integer.parseInt(portString);
+		}
+
+		String host = props.getProperty(HOST_PROPERTY);
+		if (host == null) {
+			host = DEFAULT_HOST;
+		}
+		
+		spaceNo = DEFAULT_SPACE;
+		String spaceString = props.getProperty(SPACE_PROPERTY);
+		if (spaceString != null) {
+			spaceNo = Integer.parseInt(spaceString);
+		}
+
+		try {
+			this.connection = new TarantoolConnection16Impl(host, port);
+		}
+		catch (Exception exc) {
+			System.err.println("Can't init Tarantool connection:" + exc.toString());
+			exc.printStackTrace();
```

Comment:
nit: this should use your Logger and log at FATAL rather than writing to System.err and printing a stacktrace.

(nits are suggestions that you needn't fix if you're pressed for time or you disagree)